### ISSUE TITLE
Prolly should be marked spam after but have some input for both magma and foxserver

### ISSUE THREAD

**Issue by Xplodin** [Luohuayu/FoxServer#7]
Open source by definition means you can use their code without their knowledge (even if its a copy or not i dont really care)

Its stated on https://opensource.org/osd-annotated
which it also means it cannot be restricted either. so magma cannot claim copying unless their license explicitly says otherwise

9. License Must Not Restrict Other Software
The license must not place restrictions on other software that is distributed along with the licensed software. For example, the license must not insist that all other programs distributed on the same medium must be open-source software.

Rationale: Distributors of open-source software have the right to make their own choices about their own software.

Yes, the GPL v2 and v3 are conformant with this requirement. Software linked with GPLed libraries only inherits the GPL if it forms a single work, not any software with which they are merely distributed.

Not only this, but magma is using code from Forge and Paper, so idk what their deal is

**Comment by Kotori0629** [comment]
thx

**Comment by JustRed23** [comment]
Let me just reopen this real quick, the license also stated this:

You may convey a Combined Work under terms of your choice that,
taken together, effectively do not restrict modification of the
portions of the Library contained in the Combined Work and reverse
engineering for debugging such modifications, if you also do each of
the following:

   a) Give prominent notice with each copy of the Combined Work that
   the Library is used in it and that the Library and its use are
   covered by this License.

   b) Accompany the Combined Work with a copy of the GNU GPL and this license
   document.

   c) For a Combined Work that displays copyright notices during
   execution, include the copyright notice for the Library among
   these notices, as well as a reference directing the user to the
   copies of the GNU GPL and this license document.
   
   
   **TL;DR** you need to include the license, retain the original package names (or add a notice that you changed the package name), and leave all license headers intact. You have changed the LICENSE which means that you did not follow the licensing.

**Comment by Luohuayu** [comment]
When I woke up early in the morning, I got a DMCA notice.
But that didn't do any much. it's stupid, it's ridiculous.

**Comment by JustRed23** [comment]
> This law clearly provides a framework, called the DMCA Takedown (Section 512 (c)(3)), for a process between two parties for the request and removal of copied content found online. For more information you can visit [this](www.copyright.gov/legislation/dmca.pdf).

You copied magma's code, basically threw the license away and branded it as your own thing. This can and will result in a DMCA takedown. It's pretty simple.

**Comment by Luohuayu** [comment]
How much of CatServer code was stolen by Magma and Mohist 1.12.2?
I can't count.

**Comment by JustRed23** [comment]
> How much of CatServer code was stolen by Magma and Mohist 1.12.2? I can't count.

Thanks for notifying me, seems like were going to take that down as well then.

**Comment by XionioXMaster** [comment]
How much code was allegedly stolen for various mods or data packs, I can’t even count. Nobody cares about this except Magma and Mohist

**Comment by Luohuayu** [comment]
> How much code was allegedly stolen for various mods or data packs, I can’t even count. Nobody cares about this except Magma and Mohist

I'm guessing they don't understand the point of open source at all

**Comment by Hexeption** [comment]
> > How much code was allegedly stolen for various mods or data packs, I can’t even count. Nobody cares about this except Magma and Mohist
> 
> I'm guessing they don't understand the point of open source at all

I'm guessing you don't understand a license that you have to abide by.
